File: python/easy/Excel Sheet Column Title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        title = ""
        while columnNumber > 0:
            digit = (columnNumber - 1) % 26
            title = chr(ord("A") + digit) + title
            columnNumber = (columnNumber - 1) // 26

        return title

File: go/easy/Excel Sheet Column Title.go
package easy

func convertToTitle(columnNumber int) string {
	title := ""
	for columnNumber > 0 {
		digit := (columnNumber - 1) % 26
		title = string('A'+digit) + title
		columnNumber = (columnNumber - 1) / 26
	}

	return title
}
